### PR TITLE
Documentation for include and exclude globs in tsconfig.json

### LINKED ISSUE #324: Add documentation for `include` and `exclude` with glob support
We need to add glob support to tsconfig.json in https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/tsconfig.json.md

Also please use https://github.com/Microsoft/TypeScript-Handbook/tree/release-2.0 as a base branch

### PR DESCRIPTION
**Pull request by riknoll**
Fixes #324

This PR adds documentation for the glob support in tsconfig.json. I took the explanation of glob wildcards from the [original glob PR](https://github.com/Microsoft/TypeScript/pull/5980) by @rbuckton

### CODE REVIEW

File: pages/tsconfig.json.md [hunk 1]
```diff
@@ -13,72 +13,57 @@ When input files are specified on the command line, `tsconfig.json` files are ig
...
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "noImplicitAny": true,
+        "removeComments": true,
+        "preserveConstEnums": true,
+        "outFile": "../../built/local/tsc.js",
+        "sourceMap": true
+    },
+    "files": [
```
Comment (mhegazy) [inline pages/tsconfig.json.md]: no need for files here then if you have includes

Comment (mhegazy) [inline pages/tsconfig.json.md]: possibly add two examples, one with include/exclude and one with files.

Comment (riknoll) [inline pages/tsconfig.json.md]: @mhegazy I brought back the old example for the files property

**Comment by mhegazy** [comment]
👍

**Pull request merged**